### PR TITLE
KRNL-297 dev jwt script

### PR DESCRIPTION
**Pull request by Dosexe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,3 +108,25 @@ docker compose up -d
...
+2. Generate a valid JWT with provided [generateJwt](./scripts/generateJwt.ts) script
+
+#### Generate JWT script
+
+- Script requires public-private key pair encrypted with `RS256` algorithm.
+- Put generated keys [here](./scripts/keys), default file names are `jwtRS256.key` for private key and `jwtRS256.key.pub` 
+for public key
+- Copy your public key to `.env` file as a one line string
+- Run script script
+```schell
```
Comment (kibertoad) [inline README.md:127]:
```suggestion
```shell
```

File: README.md [hunk 2]
```diff
@@ -107,3 +108,25 @@ docker compose up -d
...
+npm run generateJwt
+```
+- Use token from the console in your requests
+
+- **Remember to not commit your keys to repository**
```
Comment (kibertoad) [inline README.md:132]: let's add them to gitignore then

File: README.md [hunk 3]
```diff
@@ -107,3 +108,25 @@ docker compose up -d
...
+By default, all calls to the `node-template` app will require a valid JWT token.
+
+You have multiple options to ease your development:
+1. Comment `onRequest` hook for JWT verification [here](./src/plugins/jwtTokenPlugin.ts)
+2. Generate a valid JWT with provided [generateJwt](./scripts/generateJwt.ts) script
+
+#### Generate JWT script
+
+- Script requires public-private key pair encrypted with `RS256` algorithm.
+- Put generated keys [here](./scripts/keys), default file names are `jwtRS256.key` for private key and `jwtRS256.key.pub`
```
Comment (kibertoad) [inline README.md:123]: linking some kind of instruction for generating keys may be helpful

File: package.json [hunk 4]
```diff
@@ -25,7 +25,8 @@
     "db:update-client": "npx prisma generate",
     "spec:generate": "cross-env NODE_ENV=development ts-node -r dotenv/config scripts/generateOpenApi.ts && npm run spec:validate",
     "spec:validate": "cross-env NODE_ENV=development ts-node -r dotenv/config scripts/validateOpenApi.ts",
-    "prepublishOnly": "npm run build"
+    "prepublishOnly": "npm run build",
+    "generateJwt": "cross-env NODE_ENV=development ts-node scripts/generateJwt.ts"
```
Comment (kibertoad) [inline package.json:29]: general naming convention used for commands in template is `jwt:generate` (other than for standard npm hooks).

File: README.md (hunk 3 above)
Comment (Dosexe) [reply README.md]: fixed

File: package.json (hunk 4 above)
Comment (Dosexe) [reply package.json]: fixed

**Review by kibertoad — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Dosexe) [reply README.md]: fixed

**Review by andris-sevcenko — APPROVED** [review]

**Pull request merged**